src/ToDo.UI/Strings/fr/Resources.resw

```diff
@@ -1,5 +1,8 @@
 <root>
-  <!--This file is generated from 'Resources.l12n.xml'. Do not modify it manually.-->
+  <!--
+			This file is automatically generated from 'Resources.l12n.xml'.
```

Comment:
Just need to add the standard resources.resw header so that the file can be opened in VS 😉